Superpowered/OpenSource/SuperpoweredIOSAudioIO.h

```diff
@@ -46,7 +46,7 @@ typedef void (*resetCallback)(void);
 /// @param numberOfFrames The number of frames requested.
 /// @param samplerate The current sample rate in Hz.
 /// @param hostTime A mach timestamp, indicates when this buffer of audio will be passed to the audio output.
-typedef bool (*audioProcessingCallback) (void *clientdata, float *inputBuffer, float *outputBuffer, unsigned int numberOfFrames, unsigned int samplerate, uint64_t hostTime);
+typedef bool (*audioProcessingCallback) (void *clientdata, float *inputBuffer, float *outputBuffer, unsigned int numberOfFrames, unsigned int samplerate, uint64_t hostTime, int inputBufferStatusCode);
```

Comment:
I think you need to pass a pointer. By passing a regular int, a copy of the int is passed to the function, and then that copy is assigned within the function. Outside the function, though, the original value that was passed in remains unaffected.

Reply:
Also, I don't think you are allowed to change the signature of this function. It needs to match `AURenderCallback` because it is assigned at line 521

```  AURenderCallbackStruct callbackStruct;
    callbackStruct.inputProc = coreAudioProcessingCallback;```

Reply:
Does this compile?